odoo/addons/test_new_api/models/test_new_api.py

```diff
@@ -553,9 +553,9 @@ def _compute_baz(self):
     def _compute_line_ids(self):
         for record in self:
             if not record.foo:
-                continue
+                record.line_ids = False
             if any(line.foo == record.foo for line in record.line_ids):
-                continue
+                record.line_ids = False
```

Comment:
This is a non-readonly computed field.
The point of this was to not assign the field in some cases.